framework/src/postprocessors/SideDiffusiveFluxIntegral.C

```diff
@@ -60,11 +62,36 @@ SideDiffusiveFluxIntegralTempl<is_ad>::computeQpIntegral()
     const auto & grad_u = MetaPhysicL::raw_value(_fv_variable->adGradSln(*fi));
 
     // FIXME Get the diffusion coefficient on the face, see #16809
-    return -MetaPhysicL::raw_value(_diffusion_coef[_qp]) * grad_u * _normals[_qp];
+    return -diffusivity_gradient_product(grad_u, MetaPhysicL::raw_value(_diffusion_coef[_qp])) *
+           _normals[_qp];
   }
   else
-    return -MetaPhysicL::raw_value(_diffusion_coef[_qp]) * _grad_u[_qp] * _normals[_qp];
+    return -diffusivity_gradient_product(_grad_u[_qp],
+                                         MetaPhysicL::raw_value(_diffusion_coef[_qp])) *
+           _normals[_qp];
```

Comment:
I've tried changing this a couple times, clang won't let me

Reply:
What is the error?

Reply:
precheck whining